app/src/main/java/com/beemdevelopment/aegis/ui/AuthActivity.java

```diff
@@ -354,6 +356,7 @@ private class BiometricPromptListener extends BiometricPrompt.AuthenticationCall
         public void onAuthenticationError(int errorCode, @NonNull CharSequence errString) {
             super.onAuthenticationError(errorCode, errString);
             _bioPrompt = null;
+            _auditLogManager.addVaultUnlockFailedPasswordEvent();
```

Comment:
You probably meant to call ``addVaultUnlockFailedBiometricsEvent()`` here. We should probably only log the event if the biometrics prompt was not cancelled by the user.